### PR TITLE
LPS-119623 [Testing] Update TrafficSources tests

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**
Traffic channels changed within this ticket: https://issues.liferay.com/browse/LPS-118107.
Now the traffic channels came from Analytics Cloud and we display:
- "0" instead of a "-" in most of the cases. We only display a "-" when the endpoints fail
- The share value comes to the frontend side with one decimal.
- The traffic channel has a link (to go to the detail view) iff (if and only if) the value > 0 and there are country keywords

**Solution**
I updated the frontend tests to be aligned with the new changes. There were only backend changes in the _Story_ so the frontend tests didn't fail.

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('TrafficSources', () => {
 						],
 						helpMessage: 'Testing Help Message',
 						name: 'testing',
-						share: 30,
+						share: 30.0,
```
Comment (wincent) [inline modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js:40]: As you know, in JS we don't have integers and floats, just numbers, so these are the same (`30 === 30.0` → `true`).

**Review by wincent — APPROVED** [review]
Looks fine to me. I didn't run the tests but I assume you saw it green. Feel free to `ci:forward` once you see the bot is happy with it.

File: modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js (hunk 1 above)
Comment (dgarciasarai) [reply modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js:40]: Yeah, I know. I put the format that I received from the backend :)

**Comment by dgarciasarai** [comment]
ci:test:tango

**Comment by dgarciasarai** [comment]
ci:test:tango

**Comment by dgarciasarai** [comment]
ci:test:tango

**Comment by dgarciasarai** [comment]
ci:test:tango

**Comment by dgarciasarai** [comment]
ci:forward

**Pull request closed without merging**